File: LeetCode/LeetCode_Easy/C++/ContainsDuplicate.cpp
class Solution {
public:
    bool containsDuplicate(vector<int>& nums) {
        unordered_set<int> setter;

        for (int i = 0; i < nums.size(); i++)
        {
            if (setter.find(nums[i]) != setter.end())
                return true;
                
            setter.insert(nums[i]);

        }

        return false;
    }
};

File: LeetCode/LeetCode_Easy/C#/ContainsDuplicate.cs
class ContainsDuplicate
{
    public bool Solution(int[] nums)
    {
        var hash = new HashSet<int>();

        for (int i = 0; i < nums.Length; i++)
        {
            if (hash.Contains(nums[i]))
            {
                return true;
            }

            hash.Add(nums[i]);
        }

        return false;
    }
}